admin_manual/configuration/server/config_apps_sample_php_parameters.rst

```diff
@@ -0,0 +1,124 @@
+==========================
+Apps Config.php Parameters
+==========================
+
+This document describes parameters for apps maintained by ownCloud and are
+not part of the core system.
+
+.. note:: All keys are only valid if the corresponding app is installed and enabled.
+   You MUST copy the keys needed to the active config.php file.
+ 
+Multiple configuration files
+----------------------------
+
+ownCloud supports loading configuration parameters from multiple files.
+You can add arbitrary files ending with `.config.php` in the `config/` directory.
+
+Example:
+
+You could place your email server configuration in `email.config.php`. 
```

Comment:
inline code blocks need to be wrapped in double backticks. Single backticks are for links and references.